### ISSUE TITLE
Map not rendering, nor emitting errors when invalid gmap library is used

### ISSUE THREAD

**Issue by lajosbencz** [xkjyeah/vue-google-maps#261]
Hey there, awesome lib!

I'd just like to leave a note here so that others won't fall into the same pitfall:
If you provide an invalid gmap library, like the now obsolete geocode, the map wont render nor will it display any related message in the console.

```js
import * as VueGoogleMaps from 'vue2-google-maps'
Vue.use(VueGoogleMaps, {
  load: {
    libraries: 'geocode,places',
  }
});
```

It would be nice to see at least an error message, although I have no idea if it's even possible with gmap.

**Comment by xkjyeah** [comment]
Does the same happen if you provide the API version number?

**Comment by lajosbencz** [comment]
Yes, using **v: '3'** also produces this issue (or any version for that matter).
Don't get me wrong, this is a nuisance, it's just hard to debug **IF** you make a typo, or use obsolete gmap libs :]
